sites/footers/includes/footer.html

```diff
@@ -1,29 +1,74 @@
 <footer id="wb-info">
 {% unless page.noFooterAbout %}
+	<h2 class="wb-inv">{{ i18nText-footerAbout }}</h2>
+
+{%- if page.hasFooterContextual == "true" and site.global.contextualFooter.title[i18nText-lang] and site.global.contextualFooter.links[i18nText-lang] -%}
+	<div class="contextual">
+		<nav class="container">
+			<h3>{{ site.global.contextualFooter.title[i18nText-lang] }}</h3>
+			<ul class="list-col-xs-1 list-col-sm-2 list-col-md-3">
+				{% for link in site.global.contextualFooter.links[i18nText-lang] %}
+				<li><a href="{{ link.url }}">{{ link.text }}</a></li>
+				{% endfor %}
+			</ul>
+		</nav>
+	</div>
+{%- endif -%}
+
 	<div class="landscape">
 		<nav class="container wb-navcurr">
```

Comment:
Considering that we are updating the site footer, please take the time if we still need the "wb-navcurr" plugin on it or if the plugin need to be adapted to our new design